src/perturbations.jl

```diff
@@ -223,6 +258,20 @@ function display_mean_sd_variance(
     dynare_table(data, title)
 end
 
+function display_mean_sd_variance2(sim_results, endogenous_names, model::Model)
+    original_endo_nbr = model.original_endogenous_nbr    
+    
+    title = "SIMULATED MOMENTS"
+    data = Matrix{Any}(undef, original_endo_nbr + 1, 4)
+    data[1, :] = ["VARIABLE", "MEAN", "STD. DEV.", "VARIANCE"]
+    data[2:end, 1] = endogenous_names[1:original_endo_nbr]
+    data[2:end, 2] .= map(mean, eachcol(sim_results))
```

Comment:
The mean calculated in `display_mean_sd_variance2` differs significantly that the theoretical moments calculated in `display_mean_sd_variance` for first order solution.

Notice: the simulated 2nd order mean changes if the shock in the mod file is modified.
Michel argued that this should not be happening.